ups/product_deps

```diff
@@ -54,7 +50,10 @@ c7:gv3:prof    c7:p383b:prof
 e17:gv2:debug  e17:debug  
 e17:gv2:prof   e17:prof   
 c2:gv2:debug   c2:debug  
-c2:gv2:prof    c2:prof   
+c2:gv2:prof    c2:prof
+e15:gv1:prof   e15:prof
+e15:gv1:debug  e15:debug
```

Comment:
Can you spell this out for me? What are the `g` versions? I thought `e15` was ancient history.

Reply:
ah, these are provisional hacks and we won't want them for the long term.  Unfortunately I had to commit them though so that anybody who wanted to work on the CAFMaker could pull them down.

`e15` is indeed ancient history, but it's the versioning of the dependency chain of the current `ND_CAFMaker`, and I wasn't ready to try to upgrade its whole stack myself just so that I could build `ND_CAFMaker` against it.  So I hijacked the `gv` qualifier here to provide a distinct qualifier I could use to keep the dependencies synchronized while working.

Once `ND_CAFMaker` is up to date all this will become irrelevant.  I wish there were some way to set an expiration date for these.  but I'll commit a comment at least saying what they're here for

Reply:
comment in 9de1bda4